src/Components/MediaBrowser/index.tsx

```diff
@@ -79,13 +136,13 @@ export default function MediaBrowser() {
 	};
 
 	const renderSelectedPage = () => {
-		console.log(searchTerm);
-
 		switch (currentBrowserPage) {
 			case MediaBrowserPage.MY_MEDIA:
 				return <RenderMyMediaPage />;
 			case MediaBrowserPage.MY_IMAGES:
 				return <RenderMyMediaPage />;
+			case MediaBrowserPage.MY_AUDIO:
```

Comment:
This can possible be handled with RenderMediaPage and props. Then having the default case be the fallback for the MY_{ type } pages.